### ISSUE TITLE
[Firemonkey] GM.xmlHttpRequest doesn't work on .xml files

### ISSUE THREAD

**Issue by xelsios** [erosman/support#430]
Following simple code (copied from the help) does work.
```
GM.xmlHttpRequest({
  url: 'https://dblp.uni-trier.de/xml/',
  onload: response => {
    console.log(response.responseText);
  }
});
```
Trying to grab a .xml file does not seems to work.
```
GM.xmlHttpRequest({
  url: 'https://dblp.uni-trier.de/xml/osd.xml',
  onload: response => {
    console.log(response.responseText);
  }
});
```

**Comment by erosman** [comment]
It is because in case of `https://dblp.uni-trier.de/xml/osd.xml` the response is `XMLDocument` which is blocked in userScript context.

Let me see what I can do about it.

**Comment by erosman** [comment]
Fixed in v2.45
